recipes/pyfftlog/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "pyfftlog" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8b299e45be2e25c1facc2aa67b6895ec658e7083b098fe4fd9d81b92884d39b3
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  preserve_egg_dir: true
```

Comment:
Why is this necessary?

Reply:
I have it in my other ones, because of `conda-forge` and `setuptools_scm`, but I am not sure if it is required.

See
https://github.com/conda-forge/staged-recipes/pull/10087#discussion_r344346512

Reply:
I applied this diff:
```diff
diff --git a/recipes/pyfftlog/meta.yaml b/recipes/pyfftlog/meta.yaml
index d2d270b1a..4a89ec3b2 100644
--- a/recipes/pyfftlog/meta.yaml
+++ b/recipes/pyfftlog/meta.yaml
@@ -13,7 +13,7 @@ build:
   noarch: python
   number: 0
   script: {{ PYTHON }} -m pip install . -vv
-  preserve_egg_dir: true
+#  preserve_egg_dir: true

 requirements:
   host:
```
Then built the package and checked:
```bash
tar -xf /Users/nwani/.conda/envs/cf/conda-bld/noarch/pyfftlog-0.2.0-py_0.tar.bz2 site-packages/pyfftlog/version.py && cat site-packages/pyfftlog/version.py
# coding: utf-8
# file generated by setuptools_scm
# don't change, don't track in version control
version = '0.2.0'
```

So I believe it is not needed.

Reply:
Thanks @nehaljwani . Worth a try and we'll find out.